app/javascript/widget/helpers/actionCable.js

```diff
@@ -10,6 +10,7 @@ class ActionCableConnector extends BaseActionCableConnector {
       'conversation.typing_off': this.onTypingOff,
       'conversation.resolved': this.onStatusChange,
```

Comment:
@pranavrajs @nithindavid  do we need to retain the open and resolved event handlers for widget?